src/pipx/commands/common.py

```diff
@@ -168,22 +168,25 @@ def venv_health_check(
     if not python_path.is_file():
         return (
             VenvProblems(invalid_interpreter=True),
-            f"   package {red(bold(venv_dir.name))} has invalid interpreter {str(python_path)}",
+            f"{hazard}  package {red(bold(venv_dir.name))} has invalid "
+            "interpreter {str(python_path)}",
         )
     if not venv.package_metadata:
         return (
             VenvProblems(missing_metadata=True),
-            f"   package {red(bold(venv_dir.name))} has missing internal pipx metadata.",
+            f"{hazard}  package {red(bold(venv_dir.name))} has missing "
+            "internal pipx metadata.",
         )
     if venv_dir.name != canonicalize_name(venv_dir.name):
         return (
             VenvProblems(bad_venv_name=True),
-            f"   package {red(bold(venv_dir.name))} needs its internal data updated.",
+            f"{hazard}  package {red(bold(venv_dir.name))} needs its "
```

Comment:
Add leading white space

Reply:
So I was trying to figure this out.  This actually looks really nice on macOS.  The word "package" lines up with the other "package" words on non-hazard lines if you space it this way.  The emoji just ends up to the left of consistently-tabbed text.

Unfortunately the spacing of emoji doesn't seem to be consistent across platforms, so it kind of looks crap on Windows for example. 😞

Reply:
I'm going to try an experiment with a commit.  I won't merge until we can discuss...

Reply:
haha that was tricky but I think it actually worked. 😆